src/OnboardingSPA/steps/GetStarted/Welcome/index.js

```diff
@@ -75,8 +75,14 @@ const StepWelcome = () => {
 
 	const handleMigration = () => {
 		if ( canMigrateSite ) {
-			const updates = injectMigrationStep( allSteps, stepWelcome );
-			updateAllSteps( updates.allSteps );
+			const migrationStepExists = allSteps.some(
+				( step ) => step.path === stepSiteGenMigration.path
+			);
+
+			if ( ! migrationStepExists ) {
+				const updates = injectMigrationStep( allSteps, stepWelcome );
+				updateAllSteps( updates.allSteps );
+			}
```

Comment:
So, do we need this now?

Reply:
yes the container style and classes were not being applied without adding it in the steps